<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBarang</class>
 <widget class="QWidget" name="FormBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SImpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kode_barangLabel">
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kode_barangLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nama_barangLabel">
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nama_barangLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="satuanLabel">
      <property name="text">
       <string>Satuan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="satuanComboBox">
      <item>
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>II</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>III</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IV</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="harga_beliLabel">
      <property name="text">
       <string>Harga Beli</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="harga_beliLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="harga_jualLabel">
      <property name="text">
       <string>Harga Jual</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="harga_jualLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableBarang">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
